crates/oxide/src/extractor/bracket_stack.rs

```diff
@@ -25,6 +25,7 @@ impl BracketStack {
                 b'(' => b')',
                 b'[' => b']',
                 b'{' => b'}',
+                b'<' => b'>',
```

Comment:
Any chance we could do this for just the preprocessors and not for all uses of the bracket stack in general?

Reply:
wait nvm I misunderstood what was happening. ignore me 👍